platform-hub-api/spec/services/agents/git_hub_agent_service_spec.rb

```diff
@@ -0,0 +1,92 @@
+require 'rails_helper'
+
+describe Agents::GitHubAgentService, type: :service do
+
+  let :token do
+    'foo_token'
+  end
```

Comment:
minor stylistic thing but I think I prefer one liners for short values e.g. `let :token { 'foo_token' }`.